aries-site/src/components/cards/ContentCard.js

```diff
@@ -21,8 +21,29 @@ export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
       {...rest}
     >
       <Box gap="large">
+        {previewComponent && (
+          <Box
+            background={previewComponent.cardColor || 'background-contrast'}
```

Comment:
What does `cardColor` represent? To me, it represents the background color for the preview Box, correct? If so, could we use `background` instead of `cardColor` to be more precise on what this property holds?